py/py_cromwell_setup.py

```diff
@@ -63,10 +64,10 @@ def configure_cromwell(env, proxy_url):
      print('Updating cromwell config')
      file = f'{str(Path.home())}/.cromshell/cromshell_config.json'
      configuration = {
-        'cromwell_server': proxy_url.split("swagger/", 1)[0],
+        'cromwell_server': proxy_url.split("swagger/", 1)[0] if proxy_url else "",
         'requests_timeout': 5,
         'gcloud_token_email': env['user_email'],
-        'referer_header_url': env['referer']
+        'referer_header_url': env['leonardo_url']
```

Comment:
why this change?

Reply:
The refere_header should point to leonardo since thats where the request is generating from, using refere(bascially AOU URL) was resulting in authorization error